homework_8.py

```diff
@@ -0,0 +1,45 @@
+"""My Homework 8"""
+# "Task #1"
+
+
+def solution(*args):
+    """Check an increasing sequence"""
+    numbers = args[0]
+    index_number = 0
+    numbers_of_removing = 0
+    for i in numbers:
```

Comment:
for i in range(len(numbers))

Reply:
I don't understand why I need to use "range(len(numbers))"? I need to iterate by value not index

Reply:
if it is more convenient for you, no problem from me side